### DISCUSSION TITLE
Exception in redeem courses

### DISCUSSION THREAD

**Discussion by iamshushank0** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#316] [Q&A] [answered]
when I run the script after entering all the details I get this msg
Exception in redeem courses: Detected a Cloudflare version 2 Captcha challenge, This feature is not available in the opensource (free) version.

**Comment by fakeid30** [comment, marked as answer]
This is now solved
